.github/dependabot.yml

```
@@ -0,0 +1,13 @@
+# Set update schedule for GitHub Actions
+
+version: 2
+updates:
+
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      # Check for updates to GitHub Actions every weekday
+      interval: "daily"
+    # Specify labels for pull requests
```

```suggestion
    # Allow up to 2 open pull requests for pip dependencies
    open-pull-requests-limit: 2
    # Specify labels for pull requests
```
Default number of open dependabot Pull Requests is 5 (see https://docs.github.com/en/github/administering-a-repository/configuration-options-for-dependency-updates#open-pull-requests-limit) which might be a lot. I tend to stick with 2 on my other repos, but we can negotiate a bit on what number to go with.